### PR TITLE
Support inferior-python-mode as well as python-mode

### PR DESCRIPTION
**Pull request by djr7C4**

### CODE REVIEW

**Comment by Fuco1** [comment]
Sorry, I totally forgot about this!  Should I not merge it?

**Comment by djr7C4** [comment]
Feel free to merge.  I thought it hadn't been merged because of the failed test and wanted to free up my master branch (which I accidentally used for this PR).

File: smartparens-python.el [hunk 1]
```diff
@@ -50,7 +50,8 @@
 ;; Python has no sexp suffices.  This fixes slurping
 ;; (|sys).path.append---the dot should not travel with the closing
 ;; paren
-(add-to-list 'sp-sexp-suffix (list 'python-mode 'regexp ""))
+(--each '(python-mode inferior-python-mode)
+  (add-to-list 'sp-sexp-suffix (list it 'regexp ""))
```
Comment (Fuco1) [inline smartparens-python.el]: You are missing a closing paren here :) for the `--each`, that's why the test is failing.

**Comment by djr7C4** [comment]
Ok, it should be fixed now :)

**Comment by djr7C4** [comment]
Sorry, I messed up this PR by recreating the repository.  I'll make a new one.

**Pull request closed without merging**